tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -175,6 +190,21 @@ def misc_balances(spec: Spec):
     return balances
 
 
+def misc_balances_electra(spec: Spec):
+    """
+    Helper method to create a series of balances that includes some misc. balances for Electra.
+    Usage: `@with_custom_state(balances_fn=misc_balances, ...)`
+    """
+    if not is_post_electra(spec):
+        return misc_balances(spec)
```

Comment:
is it possible that, instead of adding new `misc_balances_electra`, we modify `misc_balances` by adding new `if is_post_electra(spec)` logic?